db/sql.h

```diff
@@ -576,6 +576,16 @@ struct sql_hist_cost {
     int64_t rows;
 };
 
+struct user {
+    int have_name;
```

Comment:
we can turn this into uint8_t since you are at it...have_password too. Also reorder so char name[] is first, int field is second, and the the uint8_t and the char passwd.